src/main/java/net/fabricmc/loom/configuration/accesswidener/AccessWidenerJarProcessor.java

```diff
@@ -77,14 +77,15 @@ public String getId() {
 	@Override
 	public void setup() {
 		LoomGradleExtension loomGradleExtension = LoomGradleExtension.get(project);
+		File awPath = loomGradleExtension.getAccessWidenerPath().getAsFile().get();
```

Comment:
Haven't tested this, but `get().getAsFile()` might be a bit more efficient since afaik there's no intermediate provider instance created then.